pkg/mimir/modules.go

```diff
@@ -1024,6 +1031,54 @@ func (t *Mimir) initUsageStats() (services.Service, error) {
 	return t.UsageStatsReporter, nil
 }
 
+func (t *Mimir) initContinuousTest() (services.Service, error) {
+	start := func(ctx context.Context) error {
+		logger := util_log.Logger
+		var err error
+
+		level.Info(logger).Log("msg", "initing continuous-test")
+		// Setting the environment variable JAEGER_AGENT_HOST enables tracing.
+		if trace, err := tracing.NewFromEnv("mimir-continuous-test", jaegercfg.MaxTagValueLength(16e3)); err != nil {
+			level.Error(logger).Log("msg", "Failed to setup tracing", "err", err.Error())
+		} else {
+			defer trace.Close()
+		}
+		level.Info(logger).Log("msg", "continuous-test setup tracing happened")
```

Comment:
This whole block should be removed. Tracing is already initialised as part of Mimir.